source/concepts/protocols_funding_withdrawal.rst

```diff
@@ -0,0 +1,168 @@
+.. SPDX-FileCopyrightText: 2021 Hyperledger
+   SPDX-License-Identifier: CC-BY-4.0
+
+.. _funding_and_withdrawal_protocols:
+
+********************************
+Funding and withdrawal protocols
+********************************
+
+Defintions of asset, funding and withdrawal:
+
+1. **Asset**: An asset can be any valid token on a given blockchain platform. For
+   example, on ethereum platform, it could be ETH, any ERC20 token or any other
```

Comment:
on other pages ethereum is written uppercase Ethereum

Reply:
I have updated the text to use upper case for the first letter, here as well. 655746a